cmd/influx/config.go

```diff
@@ -60,17 +60,28 @@ func (b *cmdConfigBuilder) cmdSwitchActiveRunEFn(cmd *cobra.Command, args []stri
 	if err != nil {
 		return err
 	}
-	b.name = args[0]
-	p0, ok := pp[b.name]
-	if !ok {
-		return &influxdb.Error{
-			Code: influxdb.ENotFound,
-			Msg:  fmt.Sprintf("name %q is not found", b.name),
+	var p0 config.Config
+	var n string
+	if args[0] == "-" {
+		p0, err = b.svc.ParsePreviousActiveConfig()
+		if err != nil {
+			return err
+		}
+		n = p0.Name
+	} else {
+		n = args[0]
+		var ok bool
+		p0, ok = pp[n]
+		if !ok {
+			return &influxdb.Error{
+				Code: influxdb.ENotFound,
+				Msg:  fmt.Sprintf("name %q is not found", b.name),
```

Comment:
would it make more sense to say `config ... not found`. Something like:

```go
Msg:  fmt.Sprintf("config %q is not found", b.name),
```

name feels a bit opaque to me 🤔